### PR TITLE
Hopefully a Working Automatic Changelog

### PR DESCRIPTION
**Pull request by Zandario**
## About The Pull Request
Puts the Changelog back into use and requires us to do nothing different.

## Credits
CitMain and BeeStation
## Changelog
:cl:
add: Automatic Changelogs
add: github workflows
del: Old records that are no longer needed
tweak: Changelog systems
tweak: Updated Changelog Header
/:cl:

### CODE REVIEW

File: .github/workflows/compile_changelogs.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        run: |
+          git config --local user.email "26690887+CitadelStationBot@users.noreply.github.com"
+          git config --local user.name "CitadelStationBot"
+          git pull origin master
+          git commit -m "Automatic changelog compile [ci skip]" -a || true
+      - name: Push
+        uses: ad-m/github-push-action@master
+        with:
+          #github_token: ${{ secrets.GITHUB_TOKEN }} Use this instead if you have unprotected branches
+          github_token: ${{ secrets.CL_TOKEN }}
```
Comment (LetterN) [inline .github/workflows/compile_changelogs.yml:36]:
```suggestion
on:
  schedule:
  - cron: "0 0 * * *"

jobs:
  compile:
    name: "Compile changelogs"
    runs-on: ubuntu-latest
    steps:
      - name: "Setup python"
        uses: actions/setup-python@v1
        with:
          python-version: '3.x'
      - name: "Install deps"
        run: |
          python -m pip install --upgrade pip
          python -m pip install pyyaml
          sudo apt-get install  dos2unix
      - name: "Checkout"
        uses: actions/checkout@v1
        with:
          fetch-depth: 25
      - name: "Compile"
        run: |
          python tools/ss13_genchangelog.py html/changelog.html html/changelogs
      - name: "Convert Lineendings"
        run: |
          unix2dos html/changelogs/.all_changelog.yml
      - name: Commit
        run: |
          git config --local user.email "action@github.com"
          git config --local user.name "Changelogs"
          git pull origin master
          git commit -m "Automatic changelog compile [ci skip]" -a || true
      - name: "Push"
        uses: ad-m/github-push-action@master
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
```

File: .github/workflows/stale_issues.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: Mark issues as stale
```
Comment (LetterN) [inline .github/workflows/stale_issues.yml:1]: kev told me not to include a pr staler on main.

File: .github/workflows/docker.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+name: Docker Image CI
```
Comment (LetterN) [inline .github/workflows/docker.yml:1]: not needed

File: .github/workflows/make_changelogs.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+name: Make changelogs
```
Comment (LetterN) [inline .github/workflows/make_changelogs.yml:1]: handled by the citbot

File: .github/workflows/stale_prs.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+name: Mark pull requests as stale
```
Comment (LetterN) [inline .github/workflows/stale_prs.yml:1]: dupe?

File: .github/workflows/stale_issues.yml (hunk 2 above)
Comment (Zandario) [reply .github/workflows/stale_issues.yml:1]: >stale_issues.yml
But yeah noted

File: .github/workflows/docker.yml (hunk 3 above)
Comment (Zandario) [reply .github/workflows/docker.yml:1]: Noted and read file name vs the other one, they had it separated, you saw what I said in discord 🤷

File: .github/workflows/stale_prs.yml (hunk 5 above)
Comment (Zandario) [reply .github/workflows/stale_prs.yml:1]: Noted and read file name vs the other one, they had it separated, you saw what I said in discord 🤷

**Pull request merged**